<?xml version="1.0"?>

<launch>
	<arg name="robotName" default="y6"/>
	<!-- SLOW -->
	<!-- arg name="kp" default="0.300"/>
	<arg name="ki" default="0.100"/>
	<arg name="kd" default="0.001"/>
	<arg name="alpha_p" default="0.1"/>
	<arg name="alpha_d" default="0.5"/> -->
	<!-- FAST -->
	<arg name="kp" default="1.2"/>
	<arg name="ki" default="0.001"/>
	<arg name="kd" default="0.4"/>
	<arg name="alpha_p" default="0.5"/>
	<arg name="alpha_i" default="0.5"/>
	<arg name="alpha_d" default="0.5"/>

	<node name="si_it2_flc_fm" pkg="controllers" type="si_it2_flc_fm" output="screen" args="$(arg kp) $(arg ki) $(arg kd) $(arg alpha_p) $(arg alpha_i) $(arg alpha_d)"/>
</launch>
